inst/tests/tests.Rraw

```diff
@@ -18532,3 +18532,15 @@ test(2260.15, inherits(tryCatch(messagef("x", class="test_message"), condition=i
 test(2260.16, inherits(tryCatch(messagef("x", class="test_message"), condition=identity), "message"))
 test(2260.17, inherits(tryCatch(packageStartupMessagef("x", class="test_psm"), condition=identity), "test_psm"))
 test(2260.18, inherits(tryCatch(packageStartupMessagef("x", class="test_psm"), condition=identity), "packageStartupMessage"))
+
+test(2261.01, setNumericRounding(0L), 0L)
+test(2261.02, getNumericRounding(), 0L)
+test(2261.03, setNumericRounding(1L), 0L)
+test(2261.04, getNumericRounding(), 1L)
+test(2261.05, setNumericRounding(2L), 1L)
+test(2261.06, getNumericRounding(), 2L)
+test(2261.07, setNumericRounding(1L), 2L)
+test(2261.08, getNumericRounding(), 1L)
+test(2261.09, setNumericRounding(0L), 1L)
+test(2261.10, getNumericRounding(), 0L)
+test(2261.11, capture.output(setNumericRounding(2L)), character(0))
```

Comment:
WDYT about `test(2261.11, setNumericRounding(2L), notOutput=".*")`?

Reply:
Upon trying this locally, this doesn't work, I'm getting:
```R
Test 2261.05 produced output but should not have:
  Expected absent (case insensitive): <<.*>>
  Observed: <<[1] 2>>
```
I think this is expected behaviour because `notOutput` calls `print` I believe

This happens with the previous version too:
```R
data.table:::test(1, getNumericRounding(), notOutput=".*")
```

Reply:
thanks for investigating, please comment the purpose of the test in that case 👎